index.html

```diff
@@ -0,0 +1,224 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="queries.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;1,100;1,300;1,400;1,700&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&family=Roboto:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
+    <title>Agropreneurs</title>
+</head>
+<body>
+    <header>
+        <div class="socials">
+            <ul>
+                <li><a href=""><img src="images/images (3).png" alt="facecbook Logo"></a></li>
+                <li><a href=""><img src="images/download.png" alt="twitter Logo"></a></li>
+                <li><a href="">Logout</a></li>
+                <li><a href="">English</a></li>
+                <li><a href="">Bini</a></li>
+            </ul>
+
+        </div>
+        <nav class="nav_items">
+            <div class="Logo">
+                <a href="index.html"><img src="images/images (4).png" alt="Website Logo"></a>
+            </div>
+            <ul class="pop-up" id="pop-up">
+                
+                <li>
+                    <a href="About.html" class="about" id="nav">About</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav1">Program</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav2">Join</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav3">Sponsor</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav4">News</a>
+                </li>
+                <li>
+                    <a href="" class="about program" id="nav5">AgCampaign</a>
+                </li>
+                <img src="images/2794770.png" alt="close button" class="img" id="close">
+            </ul>
+        </nav>
+    </header>
+    <section class="main-content">
+        <img src="images/dropdown.png" alt="hamburger button" class="dropdown" id="hamburger">
+        <div class="main">    
+            <h1>Agro-preneurship summit <br>
+            (Nigeria) 2023</h1>
+            <p>A summit aimed at developing young Agro-preneurs and 
+                providing employment to the populace. <br>
+                we are feeding the  nation.
+            </p>
+            <h2>2023.01.26(thur)~28(sat)</h2>
+            <h3>@Moshood Abiola National stadium Abuja, Nigeria.</h3>
+
+        </div>
+    </section>
+    <section class="features">
+        <h2 class="header">Main program</h2>
+        <hr class="line">
+        <div class="main-container">
+            <div>
+                <img src="images/Lecture.png" alt="Lecture icon">
```

Comment:
- [ ] Good job adding this title here :clap: , I can't pass without noticed that :+1: , however as this is the min title of the section you should use a `h1`. Kindly update that in every section of your  pages :+1:

Reply:
Kindly ignore this. Just like you mentioned, there should be only one `h1` tag in an HTML file